### ISSUE TITLE
waitUntil: 'networkidle' doesn't work with <html>HELLO</html>

### ISSUE THREAD

**Issue by GautierT** [puppeteer/puppeteer#1104]
### Steps to reproduce

**Tell us about your environment:**

* Puppeteer version: 0.12.0
* Platform / OS version: Heroku cedar-16 with https://github.com/jontewks/puppeteer-heroku-buildpack
* URLs (if applicable):

**What steps will reproduce the problem?**

```
    const browser = await puppeteer.launch({args: ['--no-sandbox', '--disable-setuid-sandbox']})
    const page = await browser.newPage()

    await page.setContent('<html>HELLO</html>')
    await page.waitForNavigation({waitUntil: 'networkidle'})
    
    const pdf_buffer = await page.pdf({
      format: 'A4',
      printBackground: true
    })

    await page.close()
    await browser.close()
   
```

1. Launch Puppeteer
2. Set `<html>HELLO</html>` as content with page.setContent()
3. Wait for `'waitUntil: 'networkidle'`
4. Print a PDF

**What is the expected result?**
Return a buffer with a PDF

**What happens instead?**
`Error: Navigation Timeout Exceeded: 30000ms exceeded`

This code is working when i pass a _normal_ html string (with charset, head, body and content)

**Comment by aslushnikov** [comment]
This is fixed in 0.13.0-alpha with the new [setContent](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagesetcontenthtml-options) method. Your code should also work now.

**Comment by GautierT** [comment]
Thanks @aslushnikov.
When this release will be available ?
Edit : i used `npm install git+https://github.com/GoogleChrome/puppeteer.git#ab9b34cd5dad54e170be916016a4e9b4097231ed`
And it's working. Thanks

**Comment by vineshhpatel** [comment]
@GautierT After following your last comment, I'm getting `Cannot find module './node6/Puppeteer'` error on heroku.

**Comment by GautierT** [comment]
@vineshhpatel : i fixed this by upgrading to node 8.8.0. So i can't really help you... :/
